src/migrations/20220628100710-edit-UserRoleCapability-table-constraint.ts

```diff
@@ -0,0 +1,100 @@
+import { DataTypes } from 'sequelize';
+import type { QueryInterface, Sequelize } from 'sequelize';
+
+interface SequelizeContext {
+  context: {
+    queryInterface: QueryInterface;
+    Sequelize: Sequelize;
+  };
+}
+module.exports = {
+  async up({ context: { queryInterface } }: SequelizeContext) {
```

Comment:
Same Q for this one...

Reply:
up handles the onDelete and onUpdate with CASCADE while the down handles onDelete and onUpdate with Restrict